tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```diff
@@ -1,7 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Are these changes to the csproj needed? I don't see them being used anywhere in the latest commit

Reply:
Ah, you're right, I needed these in the direct log submission PR to test some duck type stuff, and they were included in the rebase by accident, they can go I think

Reply:
Fixed in 8e280fbaeb36d290b1788ef949d74e4671193ff6